api/server.js

```diff
@@ -18,12 +20,27 @@ app.use(
   })
 );
 app.use(bodyParser.json());
+dotenv.config();
+
+// Connect to Mongoose Cluster and set connection variable
+mongoose.connect(
```

Comment:
If file `api/.env` does not exist, `process.env.DB_USERNAME` and `process.env.DB_PW` will be undefined, but the database connection can complete without any error. Do you want a check for those variables to be added in order to implement a fail-fast behavior?

Reply:
I have tried to implement the checks in branch `controllers-fork`, feel free to work on it and merge it into this branch when done

Reply:
This file gets created when running bash init.sh, no changes are required.

Reply:
I'm just worrying if anyone forgets to run the script

Reply:
Hmm, you have a good point. I kind of feel that's more of an issue we can deal with in developer documentation. Once we update the readme to include new instructions for the script, I don't think that there needs to be more checks.